### DISCUSSION TITLE
Can I split Akvorado into 3 different VM ?

### DISCUSSION THREAD

**Discussion by AndyZ1608** [akvorado/akvorado#831] [Q&A] [answered]
Hi Vincent, really appriciate if you reply my question.
I have an idea to split Akvorado into 3 VM for security issue, one for inlet service, one for Kafka, one for Database Clickhouse with 16vCPU and 32GB RAM each VM.
I have 500 flows/s. Do you think it 's possible?
Thank You.

**Comment by vincentbernat** [comment, marked as answer]
Yes, it's possible. That's the same kind of setup as with containers.
